enterprise/backend/test/metabase_enterprise/advanced_config/api/logs_test.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase-enterprise.advanced-config.api.logs-test
+  "Tests for /api/ee/logs endpoints"
+  (:require
+   [clojure.test :refer :all]
+   [java-time :as t]
+   [metabase.models.query-execution :refer [QueryExecution]]
+   [metabase.public-settings.premium-features-test :as premium-features.test]
+   [metabase.query-processor.util :as qp.util]
+   [metabase.test :as mt]))
+
+(def ^:private now #t "2023-02-28T10:10:10.101010Z")
+
+(def ^:private query-execution-defaults
+  {:hash         (qp.util/query-hash {})
+   :result_rows  0
+   :running_time 0
+   :native       true
+   :started_at   now
+   :context      :ad-hoc})
+
+(deftest fetch-logs-test
+  (testing "GET /api/logs/query_execution/:yyyy-mm"
+    (let [test-user :crowberto
+          user-id   (mt/user->id test-user)]
+      (mt/with-temp* [QueryExecution [qe-a (merge query-execution-defaults
+                                                  {:executor_id user-id
+                                                   :started_at  (t/minus now (t/days 2))})]
+                      QueryExecution [qe-b (merge query-execution-defaults
+                                                   {:executor_id user-id
+                                                    :started_at  (t/minus now (t/days 32))})]]
+        (premium-features.test/with-premium-features #{:advanced-config}
```

Comment:
Might be worth having a sanity-check test that the endpoint is blocked when `:advanced-config` isn't present